Makefile

```diff
@@ -10,15 +10,15 @@ build:
 
 .PHONY: types
 types:
-	protoc --go_out=. types/types.proto
+	protoc --go_out=$(SRCPATH) types/types.proto
 
 install:
 	go install
 
 example: default
 	protoc -I. -I$(SRCPATH) -I./vendor \
 		--go_out="plugins=grpc:$(SRCPATH)" --gorm_out="$(SRCPATH)" \
-		example/feature_demo/test.proto example/feature_demo/test2.proto
+		example/feature_demo/demo_types.proto example/feature_demo/point_service.proto
```

Comment:
`test` and `test2` are indeed pretty bad names, but I'm not sure `demo_types` and `point_service` are any better.

What is `point_service`, anyway? Can it be called `demo_service`?

Also, if we're going through the trouble of trying to make these actually readable by someone, we should add line-by-line documentation on each of the fields and messages in these files to demonstrate what they should be doing.

Reply:
Alright. I'll expand on the comments and go with `demo_service` instead. 
I'm also making Readme updates, you want me to add these here, or keep those for a separate PR?

Reply:
I'm not opinionated about that -- whatever makes sense to you